### DISCUSSION TITLE
Hue light status - how does it work?

### DISCUSSION THREAD

**Discussion by taenzerme** [Supergiovane/node-red-contrib-knx-ultimate#367] [General]
Hi there, 

thanks for the great library! 

I have a working KNX bridge connection and Hue bridge connection. I configured a Hue gradient light strip:

![CleanShot 2024-07-01 at 09 39 01](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/3618912/83aee923-6737-4722-9294-86ec2deabe50)

![CleanShot 2024-07-01 at 09 40 41](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/3618912/1c1e2d17-c360-4706-8a90-346681662d3f)

and the KNX device:

![CleanShot 2024-07-01 at 09 39 25](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/3618912/d7e5ecc6-68ef-4021-8862-211fbd1d07af)

I set up the group address in ETS and set a button on a Glass Push-button II Smart to switch the light.

Switching works fine. But I don't get any status of the lamp / lightstrip.

Is there some tutorial somewhere how to get the status to the group adress correctly?

Thanks,
Sebastian

**Comment by Supergiovane** [comment]
Hi
the status is sent from HUE device to the node, at the address 7/1/1
You can take a look at my playlist. https://www.youtube.com/watch?v=egRbR_KwP9I&list=PL9Yh1bjbLAYpfy1Auz6CKDfXUusgMwOQr
There is some videos about HUE.
The new documentation will be out soon. I've been very busy in developing the HUE nodes, so the official documentation is currently unavaiable.

**Reply by taenzerme** [reply]
Hey Max, thanks! 

This may be a stupid question, but do I have to create that GA 7/1/1 in ETS?

I did and connected that GA to the status of the MDT switch. But it does not update the status / read the status.

Any ideas why that is?

Thanks,
Sebastian

**Comment by Supergiovane** [comment]
Hi
you may not create the GA 7/1/1 in ETS, but you must create a KNX Device note (similar to what you've done for the GA 7/1/0), with GA 7/1/1 ;-)

**Comment by Supergiovane** [comment]
PS: i've misunderstood your question, sorry. If you must interact with real KNX devices on your BUS (your MDT for example), you must create the 7/1/1 in ETS as well. 
I think you cannot set the GA 7/1/1 in the MDT if you don't create a GA in ETS.

**Comment by taenzerme** [comment]
Hi Max, 

thanks, I tried, but it's still not working as expected.

I created the GA in ETS (7/1/0 = device, 7/1/1 = device status, 7/1/28 = device dim state.

I configured the GA in the Hue device like this:

![CleanShot 2024-07-01 at 21 53 15](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/3618912/86a2bd75-7785-48a1-a8b3-1c888c1baa76)

![CleanShot 2024-07-01 at 21 53 37](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/3618912/c85d7869-be02-4c15-bce9-c0ae90370733)

![CleanShot 2024-07-01 at 21 53 56](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/3618912/80763f29-79d4-42d7-a81a-1ea8f0d8122e)

But the GA value is alway empty:

![CleanShot 2024-07-01 at 21 54 52](https://github.com/Supergiovane/node-red-contrib-knx-ultimate/assets/3618912/a2d67de3-8791-4d92-bbeb-a9399ffff774)

Any idea?

Thanks,
Sebastian

**Comment by Supergiovane** [comment]
Hi
Take a look here https://github.com/Supergiovane/node-red-contrib-knx-ultimate/wiki/-Sample---Virtual-Device

If you intend to read the value of 7/1/1 as well as 7/1/28, the corresponding node must have the appropriate settings. You need to set the react to read telegram option, as well as “respond with” (this option appears as soon as you click “react to read requests)
I suggest you to fully understand the KNX part of the nodeset, by taking a look at the wiki.

**Comment by taenzerme** [comment]
Thanks Max, that helped a lot, but wasn' really the source of my problems.

I used a tunneling channel already in use for a different connection. Switching to a different tunneling channel of my IP gateway made it work :-)
